modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/edit_object_definition.jsp

```diff
@@ -22,34 +22,36 @@ ObjectDefinitionsDetailsDisplayContext objectDefinitionsDetailsDisplayContext =
 %>
 
 <div class="lfr-object-edit-object-definition">
```

Comment:
I've noticed that this included div is not under feature flag. Can the inclusion of this div break poshi tests?

Reply:
I do not think so. In addition, the `lfr-object-edit-object-definition` class is only loaded by the `ManagementToolbar` ([here](https://github.com/liferay-objects/liferay-portal/blob/d89e7d057343996dd8f3b7e01c0ed347cc644f5a/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ManagementToolbar.tsx#L25-L27)) which is under FF [here](https://github.com/liferay-objects/liferay-portal/blob/2d94c4df12eb31be442d611ac7c199aae32fecbe/modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/edit_object_definition.jsp#L25-L29). But I will run `ci:test:objects` to be sure.